contracts/bridge/Bridge.sol

```diff
@@ -199,18 +197,18 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
     }
 
     // requestKLAYTransfer requests transfer KLAY to _to on relative chain.
-    function requestKLAYTransfer(address _to, uint256 _fee) external payable {
-        _requestKLAYTransfer(_to, _fee);
+    function requestKLAYTransfer(address _to, uint256 _amount) external payable {
+        uint256 feeLimit = msg.value.sub(_amount);
```

Comment:
How about moving this logic to `_requestKLAYTransfer` modifying its parameter type.

Reply:
It is just trade off. :) Look 196 line. For () payable function, we need to calculate _amount. T.T

Reply:
Anyway, I will change and compare. =)

Reply:
If it is applied like below, it need one line more. :)
![image](https://user-images.githubusercontent.com/29390878/61675218-62426800-ad32-11e9-81f7-a9548d76d8f2.png)

Reply:
As we talked, both ways are similar. Let's keep this code. :) Thanks.